terra_layer/views.py

```diff
@@ -284,5 +284,7 @@ class SceneViewset(ModelViewSet):
     queryset = Scene.objects.all()
     serializer_class = SceneSerializer
 
+    permission_classes = (ScenePermission, )
+
     def get_queryset(self):
-        return self.model.objects.all()
\ No newline at end of file
+        return self.model.objects.all()
```

Comment:
You don't need to define get_queryset since queryset attribute is defined.

Reply:
Done